### ISSUE TITLE
`actionSaveSubmission` does not update `isIncomplete` flag for front end edits for multi-page forms.

### ISSUE THREAD

**Issue by leevigraham** [verbb/formie#454]
**Description**

`actionSaveSubmission` does not update `isIncomplete` flag for front end edits for multi-page forms.

**Steps to reproduce**
1. create a form with multiple pages
2. start the submission
3. close the browser
4. open a new browser
5. edit the submission on the front end
6. complete the form

### Form settings

- Multi-page form: Yes
- Submission Method: Page Reload
- Client-side Validation: No
- Custom Form Templates: No

**Additional info**
- Plugin version: 1.4.3
- Craft version: 3.6.12.1
- Multi-site: Yes or No

**Additional context**

https://user-images.githubusercontent.com/25124/116836999-0b0e3e80-ac0c-11eb-8cf3-a104248ae656.mov

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.4.3",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.4.4](https://github.com/verbb/formie/releases/tag/1.4.4)
